index.html

```diff
@@ -23,7 +23,20 @@ <h2>Properties</h2>
 
         <section class="dashboard--layouts">
           <h2>Layouts</h2>
-          <ul></ul>
+          <ul>
+          <li id="armghan-zubair" class="post-it">
+            <h3>Display inline</h3>
+            <p>
+              Inline elements do not generate a line-break before or after
+              themselves. The next element will be in the same line, as long
+              as there's enough space and no other properties stating
+              otherwise. You cannot adjust the height and width of these
+              elements.
+            </p>
+          </li>
+            </li>
```

Comment:
There are 2 closing </li> tags here without matching opening tags. These can be deleted. Sorry, I didn't spot this before.